include/nuttx/audio/audio.h

```diff
@@ -465,8 +480,8 @@ struct audio_buf_desc_s
   uint16_t            numbytes;           /* Number of bytes to allocate */
   union
   {
-    FAR struct ap_buffer_s  *pBuffer;     /* Buffer to free / enqueue */
-    FAR struct ap_buffer_s  **ppBuffer;   /* Pointer to receive allocated buffer */
+    FAR struct ap_buffer_s  *buffer;     /* Buffer to free / enqueue */
+    FAR struct ap_buffer_s  **pbuffer;   /* Pointer to receive allocated buffer */
   } u;
 };
```

Comment:
audio.h cannot be changed!  You cannot chanage audio.h without also changing all of the files that include audio.h.

You will either have to remove the changes to this (and any other common header files), or modify all places where the header file is included.  That includes apps/examples/i2sloop/i2sloop_main.c

Reply:
thanks for pointing that the apps are failing 
In the NuttX code I changed all files that depend on it and fixed the violation but i did not check the apps

I will fix it now and open another PR for apps
Thanks